### PR TITLE
UNIX updates; add cheat sheet

### PR DESCRIPTION
**Pull request by bewt85**
Probably the only controversial change is the cheat sheet which it would be nice to have a second opinion on.

### CODE REVIEW

**Comment by bewt85** [comment]
also @martinghunt I changed some of the sequence names in 1ca659b because people cheated in the last course.  Was there anywhere else you thought we should make this sort of change?

File: Notebooks/Unix/basic/basic.ipynb [hunk 1]
```diff
@@ -280,7 +273,7 @@
     "\n",
     "The `cd` command will change the current working directory to another, in other words allow you to move up or down in the directory hierarchy. \n",
     "\n",
-    "To move into the `Styphi` directory use:"
+    "To move into the `Styphi` directory type the folling.  Note, you'll rmember this more easily if you type this into the terminal rather copying and pasting.  Also remember that you can use tab completion to save typing all of it."
```
Comment (martinghunt) [inline Notebooks/Unix/basic/basic.ipynb]: typo: folling -> following

File: Notebooks/Unix/basic/basic.ipynb [hunk 2]
```diff
@@ -259,6 +241,17 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
+    "#### Tab completion\n",
+    "\n",
+    "Typing out file names is really boring and you're likely to make typos which will at best make your command fail with a strange error and at worse overwrite some of your carefully crafted analysis.  _Tab completion_ is a trick which normally reduces this risk significantly.\n",
```
Comment (martinghunt) [inline Notebooks/Unix/basic/basic.ipynb]: Typo: worse -> worst

File: Notebooks/Unix/files/files.ipynb [hunk 3]
```diff
@@ -6,7 +6,21 @@
    "source": [
     "# Looking inside files\n",
```
Comment (martinghunt) [inline Notebooks/Unix/files/files.ipynb]: Could add "less" here

File: Notebooks/Unix/cheat_sheet/unix_cheat_sheet.ipynb [hunk 4]
```diff
@@ -0,0 +1,207 @@
...
+    "ls -lah foo/ # Give the the details (-l) of all files and folders (-a) using human readable file sizes (-h)\n",
+    "cd ../.. # Move up two directories\n",
+    "cd ../foo/bar # Move up one directory and down into the foo/bar/ subdirectories\n",
+    "cp -r foo/ baz/ # Copy the foo/ directory into the baz/ directory\n",
+    "mv baz/foo .. # Move the foo directory into the parent directory\n",
+    "rm -r ../foo # remove the directory called foo/ from the parent directory\n",
+    "find foo/ -name \"*.gff\" # find all the files with a gff extension in the directory foo/\n",
+    "```\n",
+    "\n",
+    "## Looking in files\n",
```
Comment (martinghunt) [inline Notebooks/Unix/cheat_sheet/unix_cheat_sheet.ipynb]: Could add "less" here

File: Notebooks/Unix/files/files.ipynb [hunk 5]
```diff
@@ -141,9 +166,74 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
+    "## Writing to files\n",
+    "\n",
+    "So far we've been running commands and outputting the results into the terminal.  Thats obviously useful but what if you want to save the results to another file?\n",
```
Comment (martinghunt) [inline Notebooks/Unix/files/files.ipynb]: Typo: thats -> that's

File: Notebooks/Unix/files/files.ipynb [hunk 6]
```diff
@@ -141,9 +166,74 @@
...
+   "outputs": [],
+   "source": [
+    "cat first_Styphi_line.txt"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We don't need `first_Styphi_line.txt` any more so lets delete it"
```
Comment (martinghunt) [inline Notebooks/Unix/files/files.ipynb]: Typo: lets -> let's

**Comment by bewt85** [comment]
@martinghunt, I think it's ready when you are.

**Pull request merged**